bin/download-source-tables.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env npx ts-node
+import fs from 'fs'
+import path from 'path'
+
+import config from '../server/config'
+import S3Client from '../server/data/s3Client'
+import { TableType } from '../server/services/analyticsServiceTypes'
+
+const tableTypes: ReadonlyArray<string> = Object.keys(TableType).sort()
+
+function printHelpAndExit() {
+  const scriptName = path.basename(process.argv[1])
+  const tableTypeOptions = tableTypes.map(t => `"${t}"`).join(' | ')
+  process.stderr.write(
+    `Usage: ./${scriptName} [${tableTypeOptions}] [YYYY-MM-DD] [path to save to or '-' for stdout]\n` +
+      `NB: Only works on pods within Cloud Platform\n`
+  )
+  process.exit(1)
+}
+
+const [tableTypeKey, date, savePath] = process.argv.slice(2, 5)
```

Comment:
i think i might enhance this with `yargs` or similar in future, because this is more error prone